wrappers/services.go

```diff
@@ -901,33 +902,48 @@ func EnsureSnapServices(snaps map[*snap.Info]*SnapServiceOptions, opts *EnsureSn
 	return context.reloadModified()
 }
 
-// StopServicesFlags carries additional parameters for StopServices.
-// XXX: Rename to StopServicesOptions
-type StopServicesFlags struct {
-	Disable bool
-	ScopeOptions
-}
+// categorizeServices returns a list of system and user services
+// ordered by the same order that 'app' is passed into.
+func categorizeServices(apps []*snap.AppInfo, includeActivatedServices bool) ([]string, []string) {
```

Comment:
it would be good for the return value to have names as well so it's clearer what is what

Reply:
Nice suggestion